scripts/lib/CIME/SystemTests/eri.py

```diff
@@ -145,6 +145,7 @@ def run_phase(self):
 
         _helper(dout_sr1, refdate_2, refsec_2, rundir2)
 
+        self._skip_pnl = False
```

Comment:
Wondering why this is here.

Reply:
Ah - because something seems to set skip_pnl back to true after line 106 and again before line 199.   I gave up trying to figure out what was doing it and just always make sure it's false when calling run_indv

Reply:
Should I remove this change?